### DISCUSSION TITLE
BCDeshiG's Beta Map Archive Re-Pack

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#158] [Map Releases]
This is a re-pack of BCDeshiG's NZ:P Beta map archival effort after Mediafire's content pruning in late 2020. It's packaged to be much easier to mass-install all of the maps/conforming to the release standard on discussions. I am *not* responsible for any material created in this pack, this is just for preservation/ease of use purpose.

![DOWNLAOD](https://user-images.githubusercontent.com/26030250/216108945-3b0ccf77-a5a8-4eb2-b3fd-97aee2d3ffb4.gif)
# [DOWNLOAD](https://archive.org/download/nzp_beta_archive_repack/bcdeshig_beta_archive_repack.zip)

This pack does **not** contain other assets like weapon skin variants or released mapper source files. For that you can view the original map packs here:
https://drive.google.com/file/d/1nKGW-IgElYG93tE1qg_ZgURB4QeGNu7l/view
https://drive.google.com/file/d/1Vyou37aCpX4s5RPMsaoNKpxuLDsTlyGO/view

**Comment by M0THRAK** [comment]
Hey there's an issue with the download link!

**Reply by M0THRAK** [reply]
Never mind, the map download works now.
